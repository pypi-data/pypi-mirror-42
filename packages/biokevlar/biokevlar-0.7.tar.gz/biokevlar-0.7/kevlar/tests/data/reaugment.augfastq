@read1
GCTGCACCGATGTACGCAAAGCTATTTAAAACCATAACTATT
+
BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB
GCTGCACCG          11 0 0#
 CTGCACCGA          11 0 0#
@read2
CTCACTTAACAAAAGAGCTAGGCGGTTTGGTTCCCCCGCGTT
+
BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB
         CAAAAGAGC          11 0 0#
                               TCCCCCGCG          11 0 0#
