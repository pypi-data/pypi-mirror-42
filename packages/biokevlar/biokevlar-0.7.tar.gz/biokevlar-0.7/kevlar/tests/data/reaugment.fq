@read1
GCTGCACCGATGTACGCAAAGCTATTTAAAACCATAACTATT
+
##########################################
@read2
CTCACTTAACAAAAGAGCTAGGCGGTTTGGTTCCC
+
###################################
