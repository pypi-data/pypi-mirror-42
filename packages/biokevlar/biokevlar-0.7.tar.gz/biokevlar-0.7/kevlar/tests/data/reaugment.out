@read1
GCTGCACCGATGTACGCAAAGCTATTTAAAACCATAACTATT
+
##########################################
GCTGCACCG          11 0 0#
 CTGCACCGA          11 0 0#
@read2
CTCACTTAACAAAAGAGCTAGGCGGTTTGGTTCCC
+
###################################
         CAAAAGAGC          11 0 0#
